theano/sandbox/cuda/dnn_base.c

```diff
@@ -7,8 +7,9 @@ c_set_tensorNd(CudaNdarray *var, cudnnTensorDescriptor_t desc) {
 
 
   int dim = CudaNdarray_NDIM(var);
-  int strides[dim];
+  int *strides = (int *)malloc(dim * sizeof(int));
```

Comment:
@abergeron ok, is there a pattern for what should happen if it fails? Just return -1?